opencog/base/Dockerfile

```diff
@@ -0,0 +1,70 @@
+# For creating images with all the dependencies for development installed.
+#
+# docker build --no-cache -t opencog/opencog-deps .
```

Comment:
If this is called "base", then it should probably be used to build "opencog-base" not "opencog-deps"